app/webroot/test.php

```diff
@@ -77,30 +77,30 @@
  *
  */
 if (!defined('WEBROOT_DIR')) {
-	define('WEBROOT_DIR', basename(dirname(__FILE__)));
+    define('WEBROOT_DIR', basename(dirname(__FILE__)));
 }
 if (!defined('WWW_ROOT')) {
-	define('WWW_ROOT', dirname(__FILE__) . DS);
+    define('WWW_ROOT', dirname(__FILE__) . DS);
 }
 
 if (!defined('CAKE_CORE_INCLUDE_PATH')) {
-	if (function_exists('ini_set')) {
-		ini_set('include_path', ROOT . DS . 'lib' . PATH_SEPARATOR . ini_get('include_path'));
-	}
-	if (!include 'Cake' . DS . 'bootstrap.php') {
-		$failed = true;
-	}
+    if (function_exists('ini_set')) {
+        ini_set('include_path', ROOT . DS . 'lib' . PATH_SEPARATOR . ini_get('include_path'));
+    }
+    if (!include 'Cake' . DS . 'bootstrap.php') {
```

Comment:
__[staging][code_sniffer]__
* Operator ! prohibited; use === FALSE instead
* Concat operator must not be surrounded by spaces
* Concat operator must not be surrounded by spaces